packages/documentation-site/patternfly-docs/content/layouts/compass/cards/RecentActivityCard.tsx

```diff
@@ -0,0 +1,169 @@
+                onOpenChange={() => {}}
+                toggle={(toggleRef: React.Ref<MenuToggleElement>) => (
+                  <MenuToggle
+                    ref={toggleRef}
+                    variant="plain"
+                    onClick={() => {}}
+                    isExpanded={false}
+                  >
+                    <EllipsisVIcon />
+                  </MenuToggle>
+                )}
+                popperProps={{ position: 'right' }}
+              ></Dropdown>
+            ),
+            hasNoOffset: false,
+            className: '',
+          }}
+        >
+          <CardTitle>
+            <Flex alignItems={{ default: 'alignItemsCenter' }}>
+              <PortIcon />
+              <span>Recent activity</span>
+            </Flex>
+          </CardTitle>
+        </CardHeader>
+        <CardBody>
+          <Table aria-label="Recent activity table" variant="compact" className="pf-m-plain">
+            <Thead>
+              <Tr>
+                <Th width={30}>Name</Th>
+                <Th width={30}>Project</Th>
+                <Th width={30}>Progress</Th>
+                <Th width={10} />
+              </Tr>
+            </Thead>
+            <Tbody>
+              {activityData.map((activity, rowIndex) => (
+                <Tr key={activity.id}>
+                  <Td>
+                    <Button variant="link" isInline component="a" href="#">
+                      {activity.name}
+                    </Button>
+                  </Td>
+                  <Td>
+                    <Button variant="link" isInline component="a" href="#">
+                      {activity.project}
+                    </Button>
+                  </Td>
+                  <Td>
+                    <ProgressStepper isCompact>
+                      {activity.progress.map((stepVariant, stepIndex) => (
+                        <ProgressStep
+                          id={`progress-step-${rowIndex}-${stepVariant}-${stepIndex}`}
+                          key={stepIndex}
+                          variant={
+                            stepVariant as
+                              | 'default'
+                              | 'success'
+                              | 'pending'
+                              | 'danger'
+                              | 'warning'
+                              | 'info'
+                          }
+                          icon={iconMap[stepVariant as keyof typeof iconMap]}
+                          aria-label={`Step ${stepIndex + 1} is ${stepVariant}`}
+                        />
+                      ))}
+                    </ProgressStepper>
+                  </Td>
+                  <Td isActionCell>
+                    <Dropdown
+                      isOpen={false}
+                      onSelect={() => {}}
+                      onOpenChange={() => {}}
+                      toggle={(toggleRef: React.Ref<MenuToggleElement>) => (
+                        <MenuToggle
```

Comment:
Just need to add an aria-lable to the toggles, preferably unique/using the "Name" cell in the table as reference